checks/noexec

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+TARGET="/opt/puppetlabs"
+
+if findmnt --target "$TARGET" --noheadings --output options | grep -q noexec ; then
+  echo "$TARGET is on mount with the noexec option which causes problems" >&2
```

Comment:
Perhaps we can be more specific here?

```suggestion
  echo "$TARGET is on mount with the noexec option. Installation requires access to binaries which are stored under $TARGET and must therefore be mounted with exec" >&2
```

Reply:
Sounds good! Also maybe add a newline after the dot. Nobody wants to read long lines ;)

Reply:
I like it.